aikau/src/main/resources/alfresco/forms/controls/css/BaseFormControl.css

```diff
@@ -1,37 +1,31 @@
 .claro .dijitMenu {
    border-color: @standard-border-color;
 }
-
 .alfresco-forms-controls-BaseFormControl {
    margin-bottom: @standard-line-height;
 }
 
-.alfresco-forms-controls-BaseFormControl > .title-row > label {
-   font-size: @normal-font-size;
-   line-height: ceil(@normal-font-size/@standard-line-height) * @standard-line-height;
-   margin-bottom: 3px;
-   display: inline-block;
-}
-
-.alfresco-forms-controls-BaseFormControl.hiddenlabel > .title-row {
```

Comment:
You've removed the .hiddenLabel class but you've not removed all the references to it, e.g. from PdfJs.js, InlineEditProperty.js, InlineEditSelect.js, Tags.js. If we're removing the class we should probably also be removing references to it.